test/classes/initializers/inheritance/inheritedInitWrongName.prediff

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+output=$2
+cat $output | sed '/^\$CHPL_HOME\/modules\// d' > $output.tmp
```

Comment:
I'd personally write this line as: `grep -v '\$CHPL_HOME/modules/' $output > $output.tmp`.

Reply:
I copied this file from a number of other identical existing prediffs on initializer testse and while I'm a big fan of never using sed, would prefer to keep them consistent rather than have one differ (or rewrite all of them in this PR).

Reply:
(I'm interpreting this comment as feedback rather than something you thought I needed to necessarily do, so am going to proceed, but we can tackle it post-merge if I'm wrong).